content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -32,37 +32,47 @@ different approach.
 
 ## The Kubernetes network model
 
-Every `Pod` gets its own IP address. This means you do not need to explicitly
-create links between `Pods` and you almost never need to deal with mapping
-container ports to host ports.  This creates a clean, backwards-compatible
-model where `Pods` can be treated much like VMs or physical hosts from the
-perspectives of port allocation, naming, service discovery, load balancing,
-application configuration, and migration.
+Every `Pod` gets its own IP address, maximum one per IP family. This means
+you do not need to explicitly create links between `Pods` and you almost
+never need to deal with mapping container ports to host ports.  This creates
+a clean, backwards-compatible model where `Pods` can be treated much like
+VMs or physical hosts from the perspectives of port allocation, naming,
+service discovery, load balancing, application configuration, and migration.
+
+Kubernetes IP addresses exist at the `Pod` scope, in the `status.PodIPs` field
+- containers within a `Pod` share their network namespaces - including their IP
+address. This means that containers within a `Pod` can all reach each other's
+ports on `localhost`. This also means that containers within a `Pod` must
+coordinate port usage, but this is no different from processes in a VM. This is
+called the _IP-per-pod_ model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (on platforms that support it). Even if a host has
+multiple IPs, host-network pods only have one Kubernetes IP address at
+the `Pod` scope, that is, the `status.PodIPs` field will only ever contain one
+IP per address family, so the "IP-per-pod" model is guaranteed.
 
 Kubernetes imposes the following fundamental requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods on a node can communicate with all pods on all nodes without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
+   * any pod-network pod on any node can communicate with all other pod-network
+     pods on all nodes without NAT.
+   * non-pod processes on a node (the kubelet, and also for example any other system daemon) can
+     communicate with all pods on that node.
 
-Note: For those platforms that support `Pods` running in the host network (e.g.
-Linux):
+In addition, for platforms and runtimes that support running pods in the host OS' network
+network:
 
-   * pods in the host network of a node can communicate with all pods on all
-     nodes without NAT
+   * host-network pods of a node can connect directly with all pods IPs on all
```

Comment:
now that we have Windows HostProcess Containers this statement might not be correct for Windows when running in overlay (for same reasons as https://github.com/kubernetes/website/pull/28454/files#r688944775) 

I just tried it and was not able to go from hostprocess container to a pod when in overlay.  cc: @marosset @daschott

Reply:
We should probably update https://github.com/kubernetes/website/blob/f4462936596941a5eab5b3ba322a93e2d96437a3/content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md?plain=1#L542 while we are at it

Reply:
think on this as the minimum you want to enforce the ecosystem to comply with, not as the current status, i.e., _ Least Common Multiple _

Reply:
Is this confirmed now?

Reply:
@marosset  I will keep this PR focus on the networking model if you don't mind, the windows docs can be updated in a follow-up.

@jsturtevant do you mean that host-network pods on windows are not able to reach other pods in other nodes? or ir just an implementation detail of the cni plugin for windows with overlay?

Right now a lot of distros and installers deploy apiservers as hostnetwork pods, if those pods are not reachable from the rest of the pods , in-cluster config will not work, I think that we should not remove that restriction from linux, but we can relax it for windows ???

Reply:
In case it helps, we don't support running the control plane on Windows / we don't document that we support that.

Reply:
it was an example, my point is that some operators, charts, manifests assume this, if this doesn't work there has to be one version of those operators, manifest, ... per platform ...  IMHO very bad UX if we drift on something like this, we end with 2 kubernetes flavors .
I think that is important to know if this is a technical limitation on windows or an implementation detail that can be reasonable solved  .... it puzzles me that it works with one cni and not with the other